0-ad-history-mod-package/simulation/templates/structures/mace/dock.xml

```diff
@@ -10,6 +10,10 @@
   <Identity>
     <Civ>mace</Civ>
     <SpecificName>Limēn</SpecificName>
+    <History>
+        The history of Macedonian naval activities is poorly documented and often overshadowed by their achievements on land. However, under Philip II, the Macedonians developed a navy that was first assembled by him and continued to be expanded under the Antigonid dynasty. Alexander the Great relied on Macedonian and Greek allied fleets on several occasions during his military campaign against Persia to secure the sea lanes and transport supplies and reinforcements. He used 160 triremes from his Greek allies to transport his army across the Dardanelles at the start of his campaign. He also employed naval forces during the Siege of Miletus in 334 BC and the Siege of Tyre in 332 BC, where his fleets played a decisive role in capturing the cities for the Macedonians. Alexander later disbanded his Greek allied fleet since it was too expensive and no longer useful in his further campaigns.
```

Comment:
```suggestion
        The history of Macedonian naval activities is poorly documented and often overshadowed by their achievements on land. However, under Philip II, the Macedonians developed a navy and it continued to be expanded under the Antigonid dynasty. Alexander the Great relied on Macedonian and Greek allied fleets on several occasions during his military campaign against Persia to secure the sea lanes and to transport supplies and reinforcements. He used 160 triremes from his Greek allies to transport his army across the Dardanelles at the start of his campaign. He also employed naval forces during the Siege of Miletus in 334 BC and the Siege of Tyre in 332 BC, where his fleets played a decisive role in capturing the cities for the Macedonians. Alexander later disbanded his Greek allied fleet, since it was too expensive and no longer useful in his further campaigns.
```